GrocifyApp.UI/src/pages/Products/Products.module.scss

```diff
@@ -0,0 +1,67 @@
+@import "../../styles/core/variables/_colors-var.scss";
+
```

Comment:
<img width="320" alt="image" src="https://github.com/Catarina98/Grocify/assets/154355241/4a9ed37e-3c20-47c6-a334-f8039994c3b6">

The color of the icons are incorrect. See figma to see the color for disable and active section icon